browser/ui/sidebar/sidebar_web_panel_controller.h

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_SIDEBAR_SIDEBAR_WEB_PANEL_CONTROLLER_H_
+#define BRAVE_BROWSER_UI_SIDEBAR_SIDEBAR_WEB_PANEL_CONTROLLER_H_
+
+#include "base/memory/raw_ref.h"
+
+class BrowserView;
+class BraveMultiContentsView;
+
+namespace sidebar {
+
+struct SidebarItem;
+
+class SidebarWebPanelController {
+ public:
+  explicit SidebarWebPanelController(BrowserView& browser);
+  ~SidebarWebPanelController();
+
+  void OpenWebPanel(const SidebarItem& item);
+  void CloseWebPanel();
+  bool IsShowingWebPanel() const;
+
+ private:
+  BraveMultiContentsView* GetMultiContentsView();
+  BraveMultiContentsView* GetMultiContentsView() const;
```

Comment:
Should the return type be `const` too?

```suggestion
  const BraveMultiContentsView* GetMultiContentsView() const;
```

Reply:
Updated.